root.go

```diff
@@ -145,15 +144,17 @@ func (r Root) DeletePcap(pcappath string) error {
 	if err != nil {
 		return err
 	}
-	pcappath = filepath.Clean(pcappath)
+	pcappath, err = filepath.Abs(pcappath)
+	if err != nil {
+		return err
+	}
```

Comment:
Nit: Do this before the call to `r.Pcaps` so it doesn't split the definition of `files` from its use.